### ISSUE TITLE
A LOAD of new stuff needed.

### ISSUE THREAD

**Issue by testitem** [DerpfacePython/Ways-to-NOT-make-money#107]
Comparison between regular Cookie Clicker and Ways to Not Make Money:

Cookie Clicker|Ways To Not Make Money
-------------------|-------------------------------------
14 buildings|5 buildings
319 upgrades|9 upgrades
251 achievements|0 achievements

We should definitely add these things in 1.0.0 - who would want to play the game if the game has less features than Cookie Clicker?

**Comment by joshgit** [comment]
We gotta make this thing a web app.

**Comment by testitem** [comment]
Yeah... but how? JavaScript? Cookie Clicker is 98.979% JavaScript ( I made that percentage up)

**Comment by joshgit** [comment]
Read, study and grow, because achievement can be had at any age.

**Comment by DerpfacePython** [comment]
Yeah, that's true. We shouldn't really compare this game with Cookie Clicker, though, @testitem, as that game had years of work done to it (while we only had around a year or so).

**Comment by testitem** [comment]
We could save this for later. Just keep the issue open
